src/pe_reports/data/data_schema.sql

```diff
@@ -1,138 +1,226 @@
-    PRIMARY KEY (organizations_uid)
+--
+-- Name: Users; Type: TABLE; Schema: public; Owner: pe
+--
+
+CREATE TABLE public."Users" (
+    id uuid NOT NULL,
+    email character varying(64),
+    username character varying(64),
+    admin integer,
+    role integer,
+    password_hash character varying(128),
+    api_key character varying(128)
 );
 
--- Organization's Root Domains Table
-CREATE TABLE IF NOT EXISTS public.root_domains
-(
-    root_domain_uid uuid default uuid_generate_v1() NOT NULL,
+--
+-- Name: alerts; Type: TABLE; Schema: public; Owner: pe
+--
+
+CREATE TABLE public.alerts (
+    alerts_uid uuid DEFAULT public.uuid_generate_v1() NOT NULL,
+    alert_name text,
+    content text,
+    date date,
+    sixgill_id text,
+    read text,
+    severity text,
+    site text,
+    threat_level text,
+    threats text,
+    title text,
+    user_id text,
+    category text,
+    lang text,
     organizations_uid uuid NOT NULL,
-    organization_name text NOT NULL,
-    root_domain text NOT NULL,
-    ip_address text,
-    UNIQUE(root_domain, organizations_uid),
-    PRIMARY KEY (root_domain_uid)
+    data_source_uid uuid NOT NULL
 );
 
--- Organization's Sub Domains Table
-CREATE TABLE IF NOT EXISTS public.sub_domains
-(
-    sub_domain_uid uuid default uuid_generate_v1() NOT NULL,
-    sub_domain text NOT NULL,
-    root_domain_uid uuid NOT NULL,
-    root_domain text NOT NULL,
-    UNIQUE(sub_domain, root_domain_uid),
-    PRIMARY KEY (sub_domain_uid)
+
+--
+-- Name: alias; Type: TABLE; Schema: public; Owner: pe
+--
+
+CREATE TABLE public.alias (
+    alias_uid uuid DEFAULT public.uuid_generate_v1() NOT NULL,
+    organizations_uid uuid NOT NULL,
+    alias text NOT NULL
 );
 
--- Organization's Sub Domains web_assets Link Table
-CREATE TABLE IF NOT EXISTS Sub_domains_Web_assets
-(
-    sub_domain_uid uuid NOT NULL,
-    asset_uid uuid NOT NULL,
-    PRIMARY KEY (sub_domain_uid, asset_uid)
+
+--
+-- Name: asset_headers; Type: TABLE; Schema: public; Owner: pe
+--
+
+CREATE TABLE public.asset_headers (
+    _id uuid DEFAULT public.uuid_generate_v1() NOT NULL,
+    organizations_uid uuid NOT NULL,
+    sub_url text NOT NULL,
+    tech_detected text[] NOT NULL,
+    interesting_header text[] NOT NULL,
+    ssl2 text[],
+    tls1 text[],
+    certificate json,
+    scanned boolean,
+    ssl_scanned boolean
 );
 
--- Organization's IPs Table
-CREATE TABLE IF NOT EXISTS public.web_assets
-(
-    asset_uid uuid default uuid_generate_v1() NOT NULL,
-    asset_type text Not NULL,
-    asset text NOT NULL,
-    ip_type text,
-    verified boolean,
+--
+-- Name: credential_breaches; Type: TABLE; Schema: public; Owner: pe
+--
+
+CREATE TABLE public.credential_breaches (
+    credential_breaches_uid uuid DEFAULT public.uuid_generate_v1() NOT NULL,
+    breach_name text NOT NULL,
+    description text,
+    exposed_cred_count bigint,
+    breach_date date,
+    added_date timestamp without time zone,
+    modified_date timestamp without time zone,
```

Comment:
What is the reason for storing timestamps without a time zone?  Generally, that will get you into trouble at some point.

Reply:
That is a very good catch. Those columns are set to just timestamp in our database but the schema-dump, using pg_dump, changed it for some reason. I'm going to make those adjustments and double-check the rest.

Reply:
Looks like you remedied this in https://github.com/cisagov/pe-reports/pull/126/commits/8564cb8b0300091c3e7da1bb918d87aa65ca5dcc. 👍